.github/workflows/export_docs.yml

```diff
@@ -0,0 +1,27 @@
+name: Doc Export
+
+on:
+  push:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  pushdocs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: extract readmes
+        run: node scripts/extract_readmes.js
+      - name: Pushes to another repository
+        uses: cpina/github-action-push-to-another-repository@main
```

Comment:
As of at least a couple months ago, not all github actions are supported. I don't have permissions to verify which actions we have available anymore (I think).

So it will require a project admin to add this otherwise this action will fail.